test/functional/postalcode/postalcode.js

```diff
@@ -0,0 +1,74 @@
+define([
+    "globalize",
+    "json!cldr-data/supplemental/likelySubtags.json",
+    "json!cldr-data/supplemental/postalCodeData.json",
+    "../../util",
+
+    "globalize/postalcode"
+], function( Globalize, likelySubtags, postalcode, util ) {
+
+    function extraSetup() {
+        Globalize.load( postalcode );
+    }
+
+    QUnit.module( ".validatePostalCode( value )", {
+        setup: function() {
+            Globalize.load( likelySubtags, {
+                main: {
+                    ko: {}
+                }
+            });
+            Globalize.locale( "ko" );
+        },
+        teardown: util.resetCldrContent
+    });
+
+    QUnit.test( "should validate parameters", function( assert ) {
+        util.assertParameterPresence( assert, "value", function() {
+            Globalize.validatePostalCode();
+        });
+
+        util.assertStringParameter( assert, "value", function( invalidValue ) {
+            return function() {
+                Globalize.validatePostalCode( invalidValue );
+            };
+        });
+    });
+
+    QUnit.test( "should validate CLDR content", function( assert ) {
+        util.assertCldrContent( assert, function() {
+            Globalize.validatePostalCode("427-090");
+        });
+    });
+    QUnit.test( "should return boolean", function( assert ) {
+        extraSetup();
```

Comment:
This is used just once, inline the method (put the `Globalize.load` call in here directly).